### PR TITLE
changes of Motor neurons

### PR DESCRIPTION
**Pull request by Adel-Laali**

### CODE REVIEW

**Review by Nefcanto — CHANGES_REQUESTED** [review]
Hi Adel, great work. The content is good. Simplification is really good. However, you're still using HTML. Please use Markdown. In case you have problems, let me know or I can connect to you to guide you. Fix that and this request would be pulled.

Thanks.

**Comment by Adelll21** [comment]
> Hi Adel, great work. The content is good. Simplification is really good. However, you're still using HTML. Please use Markdown. In case you have problems, let me know or I can connect to you to guide you. Fix that and this request would be pulled.
> 
> Thanks.

Hi Saeid, Thanks For your helpful guide. But I should say that I use Markdown and I didn't use HTML, please tell me more about that. Also About my little problem, I understood that how can I use dash, hashtag and another tools. Therefore, I didn't know that what was you meant about "Fix that". please clear that phrase.

**Comment by Nefcanto** [comment]
@Adelll21 

This is the [link of the file in your repository](https://github.com/Adelll21/Path/blob/master/contents/path/design/motor-neuron.md). Please go to the edit mode. What you see there is not markdown. It's HTML. Let me know if you need help in changing it into markdown.

**Comment by Adelll21** [comment]
> @Adelll21
> 
> This is the [link of the file in your repository](https://github.com/Adelll21/Path/blob/master/contents/path/design/motor-neuron.md). Please go to the edit mode. What you see there is not markdown. It's HTML. Let me know if you need help in changing it into markdown.

I see. This is the link that I used for changing the text, [https://www.tablesgenerator.com/markdown_tables]. I haven't understood what was you meant about using HTML or Markdown. I definitely need your help, Please help me to do it right.

Thanks for your time.

**Comment by Adelll21** [comment]
Hi Saeid, I've done the changes. please check.

**Review by Nefcanto — CHANGES_REQUESTED** [review]
Thank you so much for this work.

However, you're overusing tables. 

Those are simple lists. Tables are for when we need to have more than one column.

Please replace tables with simple lists.

Thank you.

File: contents/path/design/blood-and-car-fluids.md [hunk 1]
```diff
@@ -1,27 +1,25 @@
...
-<p>خوب، کدومش مهندسی عظیم تری داره؟ اتومبیل که توش برای هر کار یه لوله کشی جداست و یه مایع جدا، یا بند ناف که توش برای همه کارها یه لوله کشی و یه مایع استفاده شده؟</p>
-<p>اگه میخواین عظمت این طراحی رو درک کنین، کافیه در نظر بگیرین که اگه ماشین قرار بود مثل ناف طراحی بشه، باید ما یه مایعی اختراع میکردیم که راننده ها بزنن تو باک، و اون مایع هم سوخت رسانی میکرد، هم روغن کاری و کاهش اصطکاک، هم دفع ضایعات ناشی از کار موتور، هم تمیز کردن شیشه، و هم فشار هیدرولیک برای ترمز رو فراهم میکرد.</p>
+| Car         | Function                          |
+|-------------|-----------------------------------|
+| Gasoline    | Power Supply                      |
+| Engine oil  | Reduce Friction & Engine Cooling  |
+| Water       | Glass cleaning                    |
+| Brake fluid | Hydraulic Braking Effect          |
+    
+#### Point: These Fluids Are Seperate
```
Comment (Nefcanto) [inline contents/path/design/blood-and-car-fluids.md:17]: I guess it's better to say:

Separate fluids, separate pumping, separate piping

File: contents/path/design/blood-and-car-fluids.md [hunk 2]
```diff
@@ -1,27 +1,25 @@
...
+
+<hr />
```
Comment (Nefcanto) [inline contents/path/design/blood-and-car-fluids.md:19]: To place a line in markdown, simply use three stars:

***

please don't use HTML. `<br />` is HTML tag.

File: contents/path/design/cns-pns-government.md [hunk 3]
```diff
@@ -1,14 +1,8 @@
...
-</p>
-
-<p>
-    چطور اولی رو ما طراحی کردیم، با اون همه نواقصش، ولی دومی که بسیار کامل تره تصادفی ایجاد شده؟
-</p>
+| Government                  | CNS                         |
+|-----------------------------|-----------------------------|
+| Member Management           | Member Management           |
+| Command Center              | Command Center              |
+| Handling and Fixing Defects | Handling and Fixing Fefects |
```
Comment (Nefcanto) [inline contents/path/design/cns-pns-government.md:7]: misspelling here. I recommend that you install "Grammarly" for chrome. It's an awesome extension that checks for misspellings.

Also, CNS + PNS

Also, we might change those items in the future. But for now, they are good.

File: contents/path/design/coaxial-cable-and-esophagus.md [hunk 4]
```diff
@@ -1,28 +1,23 @@
-﻿<h1>کابل و مری</h1>
+# Cable & Esophagus
+
+### Cable Picture:
 
-<p>
-    این یه عکس از یه کابل coaxial:
-</p>
 <img src="https://www.electricaltechnology.org/wp-content/uploads/2015/02/Why-Coaxial-Cables-are-Highly-Insulated.jpg" alt="Coaxial cable" />
```
Comment (Nefcanto) [inline contents/path/design/coaxial-cable-and-esophagus.md:5]: Use markdown for images.
This is the formula:

title

File: contents/path/design/coaxial-cable-and-esophagus.md [hunk 5]
```diff
@@ -1,28 +1,23 @@
...
+
+### Both are made up of different layers. Each layer covers the previous layer and has a specific function.
+
+<hr />
+
+#### For Quick Search:
+
+- Esophagus Cross Section (For Esophagus)
+
+- Cable Cross Section (For Cable)
```
Comment (Nefcanto) [inline contents/path/design/coaxial-cable-and-esophagus.md:23]: It's nice that you have deleted the **judgemental** part. Thank you.

File: contents/path/design/motor-neuron.md [hunk 6]
```diff
@@ -1,13 +1,44 @@
-﻿<h1>نورون حرکتی</h1>
+# Motor Neurons
```
Comment (Nefcanto) [inline contents/path/design/motor-neuron.md:1]: We have a problem here. Both you and the architect worked on this file. So for this reason, I recommend that we go, one by one.

**Review by Nefcanto — CHANGES_REQUESTED** [review]
Thank you Adel. 

This is the spirit we need for teamwork. 

However, to keep stuff simple, let's fix one file at a time.

**Comment by Adel-Laali** [comment]
> Thank you so much for this work.
> 
> However, you're overusing tables.
> 
> Those are simple lists. Tables are for when we need to have more than one column.
> 
> Please replace tables with simple lists.
> 
> Thank you.

Thank you for all of your useful and helpful comments.
About using too many tables, I should say that I suppose we're going to use tables instead long and judgemental sentences. Thanks for remembering. I consider that to use list for one column and use tables for more than one column.

**Comment by Adel-Laali** [comment]
> Thank you Adel.
> 
> This is the spirit we need for teamwork.
> 
> However, to keep stuff simple, let's fix one file at a time.

I got that we must make phases and sentences too simple. I'll consider that.
Just tell me what will we fix?

File: contents/path/design/blood-and-car-fluids.md (hunk 1 above)
Comment (Adel-Laali) [reply contents/path/design/blood-and-car-fluids.md:17]: Yes, I agree with you. I'll fix this.

File: contents/path/design/blood-and-car-fluids.md (hunk 2 above)
Comment (Adel-Laali) [reply contents/path/design/blood-and-car-fluids.md:19]: I see. I don't know what do I use that it's not HTML😅. Please send me a Text including HTML words or whatever else. Please send in Instagram.

Thanks

File: contents/path/design/cns-pns-government.md (hunk 3 above)
Comment (Adel-Laali) [reply contents/path/design/cns-pns-government.md:7]: I didn't find out that what was your meant. Please make it clear.

File: contents/path/design/coaxial-cable-and-esophagus.md (hunk 4 above)
Comment (Adel-Laali) [reply contents/path/design/coaxial-cable-and-esophagus.md:5]: I see.
But I don't know how do I use markdown tools. I just know a little about it.

File: contents/path/design/coaxial-cable-and-esophagus.md (hunk 5 above)
Comment (Adel-Laali) [reply contents/path/design/coaxial-cable-and-esophagus.md:23]: Yes. We have to explain our thoughts and than people choose which one is better. It's their choice.

File: contents/path/design/motor-neuron.md (hunk 6 above)
Comment (Adel-Laali) [reply contents/path/design/motor-neuron.md:1]: I'm totally surprised!!!
I Told my part in our Instagram group. I wrote the address that I have choose.
I remember that architect chose Stop-codon or something.
However, I'll fix the parts that you have told than Tell me that it's get right or not.

Thanks for your time.

**Review by Nefcanto — APPROVED** [review]
awesome, thank you

**Pull request merged**